gengokit/httptransport/httptransport.go

```diff
@@ -159,6 +187,43 @@ func (b *Binding) PathSections() []string {
 	return rv
 }
 
+func (f *Field) DecodeLogic() (string, error) {
```

Comment:
This function name, at a glance, leads me to believe that it is decoding something. However it is actually rendering something. This is complicated by the fact that it is used inside a template, whereupon it is treated like a property more than a function proper, in which case the noun-like naming makes sense.

Because of this, it seems okay to leave the naming suggesting that it's a property, although it smells a little bit to me. Perhaps more relevantly, I don't feel like "decode" is the right word here, as query string is not strictly an *encoding* of something, like, for example, base64. I suggest a name like `QueryStringUnmarshaler`.